apps/journeys-admin/src/components/TemplateView/TemplatePreviewTabs/TemplateCardPreview/TemplateCardPreview.tsx

```diff
@@ -0,0 +1,135 @@
+import 'swiper/swiper.min.css'
+
+import Box from '@mui/material/Box'
+import { Theme } from '@mui/material/styles'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { ReactElement } from 'react'
+import SwiperCore, { A11y, Mousewheel } from 'swiper'
+import { Swiper, SwiperSlide } from 'swiper/react'
+
+import { TreeBlock } from '@core/journeys/ui/block'
+import { BlockRenderer } from '@core/journeys/ui/BlockRenderer'
+import { useJourney } from '@core/journeys/ui/JourneyProvider'
+import { getJourneyRTL } from '@core/journeys/ui/rtl'
+import { ThemeProvider } from '@core/shared/ui/ThemeProvider'
+
+import {
+  GetJourney_journey_blocks_CardBlock as CardBlock,
+  GetJourney_journey_blocks_StepBlock as StepBlock
+} from '../../../../../__generated__/GetJourney'
+import { ThemeMode, ThemeName } from '../../../../../__generated__/globalTypes'
+import { CardWrapper } from '../../../CardPreview/CardList/CardWrapper'
+import { VideoWrapper } from '../../../Editor/Canvas/VideoWrapper'
+import { FramePortal } from '../../../FramePortal'
+
+SwiperCore.use([Mousewheel, A11y])
+
+interface TemplateCardPreviewProps {
+  steps?: Array<TreeBlock<StepBlock>>
+}
+
+interface TemplateCardPreviewItemProps {
+  step: TreeBlock<StepBlock>
+}
+
+export function TemplateCardPreview({
+  steps
+}: TemplateCardPreviewProps): ReactElement {
+  const smUp = useMediaQuery((theme: Theme) => theme?.breakpoints?.up('sm'))
+
+  function TemplateCardPreviewItem({
+    step
+  }: TemplateCardPreviewItemProps): ReactElement {
+    const { journey } = useJourney()
+    const { rtl, locale } = getJourneyRTL(journey)
+    const cardBlock = step.children.find(
+      (child) => child.__typename === 'CardBlock'
+    ) as TreeBlock<CardBlock>
+    return (
+      <Box
```

Comment:
Use stack since you are positioning the component inside
```suggestion
      <Stack
```